mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoInsertMethodGenerator.java

```diff
@@ -122,6 +124,17 @@ public DaoInsertMethodGenerator(
         BoundStatementBuilder.class,
         statementName);
 
+    if (attributes != null) {
+      insertBuilder.addStatement(
+          "boundStatementBuilder = populateBoundStatementWithAttributes(boundStatementBuilder, $S, $S, $S, $L, $L, $L, $S)",
+          attributes.ExecutionProfileName(),
+          attributes.consistencyLevel(),
+          attributes.serialConsistencyLevel(),
+          attributes.idempotence(),
+          attributes.pageSize(),
+          attributes.timeout(),
+          attributes.routingKeyspace());
+    }
     if (statementAttributesParam != null) {
```

Comment:
should the `statementAttributesParam` programmatic values override those specified by the annotation?
Maybe we should log a warning if both mechanisms are used on the same method?

Reply:
The statementAttributesParam should definitely override the Attributes set at compile time, provided they are being explictly set. As Olivier mentioned above right now we have  aproblem where the defaults of statementAttributesParam will override ones that are explictly set by Attributes, which is the real problem here.  


Here is how I think it should work.
1. Attributes are applied, if they aren't specific defaults are used.
2. statementAttributes are applied... their defaults should not override Specific Attributes that are set.